lib/go-tc/deliveryservices.go

```diff
@@ -335,6 +349,46 @@ func ParseOrgServerFQDN(orgServerFQDN string) (*string, *string, *string, error)
 	return &protocol, &FQDN, port, nil
 }
 
+func (ds *DeliveryServiceV31) Sanitize() {
```

Comment:
I think we might not need this if we alias `type DeliveryServiceV30 DeliveryServiceV31` like I mention in the previous comment. But if we don't want to do that for some reason, I think the matching `DeliveryServiceV30` methods would no longer be in use and could be removed. We should only be calling `Sanitize`, `Validate`, etc. in the _latest_ handlers.